src/riak_core_util.erl

```diff
@@ -876,5 +930,16 @@ proxy_spawn_test() ->
         ok
     end.
 
+-ifdef(EQC).
+
+prop_any_bin_consult() ->
```

Comment:
I think I will probably remove it, the test is in riak_kv_vnode_status_mgr. Maybe the code should be too, actually, since it addresses that specific problem.